src/app/app.ts

```diff
@@ -65,18 +65,18 @@ client.login(process.env.DISCORD_BOT_TOKEN).catch(Log.error);
 function initializeClient(): Client {
 	const clientOptions: ClientOptions = {
 		intents: [
-			Intents.FLAGS.GUILDS,
-			Intents.FLAGS.GUILD_BANS,
-			Intents.FLAGS.GUILD_MEMBERS,
-			Intents.FLAGS.GUILD_EMOJIS_AND_STICKERS,
-			Intents.FLAGS.GUILD_VOICE_STATES,
-			Intents.FLAGS.GUILD_PRESENCES,
-			Intents.FLAGS.GUILD_MESSAGES,
-			Intents.FLAGS.GUILD_MESSAGE_REACTIONS,
-			Intents.FLAGS.DIRECT_MESSAGES,
-			Intents.FLAGS.DIRECT_MESSAGE_REACTIONS,
+			"Guilds",
```

Comment:
these should look like this `GatewayIntentBits.Guilds` I believe

Reply:
I will definitely check that out and get back to you

Reply:
yes this is the format for the intents in v14.  GatewayIntentBits.Guilds, GatewayIntentBits.GuildMembers etc.